tests-foundry/common/BaseTest.t.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache2.0
+pragma solidity 0.8.4;
+
+import {console} from "forge-std/console.sol";
+
+import {System} from "../../contracts/System.sol";
+import {Test} from "forge-std/Test.sol";
+import {Deployer} from "../../scripts-foundry/Deployer.s.sol";
+
+abstract contract BaseTest is System, Test {
```

Comment:
All future test files should derive from this base. Also note the xxx.t.sol & xxx.s.sol for test contracts and script contracts respectively